tests/components/deako/conftest.py

```diff
@@ -0,0 +1,38 @@
+"""deako session fixtures."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homeassistant.components.deako.const import DOMAIN
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        data={},
+        unique_id="aabbccddeeff",
+    )
+
+
+@pytest.fixture(autouse=True)
+def deako_mock_async_zeroconf(mock_async_zeroconf: MagicMock):
+    """Auto mock zeroconf."""
```

Comment:
What's this for?

Reply:
Looks like I never used it. Not needed anyway. Thanks!